subsys/bluetooth/controller/hci/hci.c

```diff
@@ -207,6 +207,8 @@ static uint64_t le_event_mask = DEFAULT_LE_EVENT_MASK;
 static struct net_buf *cmd_complete_status(uint8_t status);
 
 #if defined(CONFIG_BT_CTLR_ADV_EXT)
+#define BUF_GET_TIMEOUT K_SECONDS(10)
```

Comment:
What is the rationale behind using 10 seconds? 

Just an arbitrary value, or is to to conform to some value in the core spec or to be similar to the `HCI_CMD_TIMEOUT` from the host?

Reply:
It is inspired by `HCI_CMD_TIMEOUT` and is arbitrary to catch any buffer leaks which otherwise without a timeout would be not noticed.

Reply:
OK thanks :)